### PR TITLE
fix: window.open not accepting size values with "px" at the end

### LINKED ISSUE #26077: window.open does not accept size values with "px" at the end
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 10.1.4
* **Operating System:**
  * Windows
* **Last Known Working Electron version:**
  * 9.3.2

### Expected Behavior
The window should open with the correct size

### Actual Behavior
The size is ignored

### To Reproduce
Use nativeWindowOpen and do: window.open("", "", "width=300px")

### Additional Information
This is working in chrome and firefox and was working in electron 9. If the event is intercepted via webContents.on("new-window") then the value is shown as NAN,

### PR DESCRIPTION
**Pull request by Code-Blooded-Human**
#### Description of Change
Close #26077 , correctly parses "px" at end of size string
#### Checklist
- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes
notes: Fix `window.open` not accepting size values with "px" at the end.

### CODE REVIEW

**Review by codebytere — COMMENTED** [review]
@Code-Blooded-Human please correct your commit to be semantic (the commit _itself_ must start with `fix`) - this also needs proper release notes and a test as well. On the original issue, you can see that if the event is intercepted via `webContents.on("new-window")` then the value is shown as NaN - so a good test here would verify that does _not_ occur after this fix.

File: lib/common/parse-features-string.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ const keysOfTypeNumber = ['top', 'left', ...Object.keys(keysOfTypeNumberCompileT
 type CoercedValue = string | number | boolean;
 function coerce (key: string, value: string): CoercedValue {
   if (keysOfTypeNumber.includes(key)) {
-    return Number(value);
+    return parseInt(value);
```
Comment (MarshallOfSound) [inline lib/common/parse-features-string.ts:41]:
```suggestion
    return parseInt(value, 10);
```

The radix does not default to 10

**Review by zcbenz — APPROVED** [review]

**Comment by pushkin-** [comment]
will this be backported to 10?

**Pull request merged**